scalyr_agent/copying_manager/copying_manager.py

```diff
@@ -435,7 +445,13 @@ def keys(self):
 
     def pop(self, path, worker_id, default=None):
         # type: (six.text_type, six.text_type, Optional[object]) -> Optional[object]
-        return self.__paths[path].pop(worker_id, default)
+
+        value = self.__paths[path].pop(worker_id, default)
+
+        if not self.__paths[path]:
```

Comment:
Similarly this could be `if path in self.__paths and not self.__paths[path]:`

Reply:
It will work because of defaultdict, but it's confusing, I will change it. Thanks!